procurement.md

```diff
@@ -0,0 +1,695 @@
+A vendor who merely delivers a timely set of open source components
+has not actually done enough to succeed at contributing to a
+successful open source project.  In addition to writing quality open
+source code, the vendor must enable all the other teams to succeed by
+participating in and reinforcing the open source process.
+
+Some vendors will be unable to manage this participation.  They won't
+have the open source experience or, in some cases, the temperament.
+No matter how much an agency might prefer experienced open source
+vendors during the procurement process, it is likely that at least
+some vendors will end up on the team because they submitted bids that
+scored high on non-open source criteria.  To succeed, an agency cannot
+assume all its vendors will have the open source skills they need.
+The agency can instead provide structures that ensure success even
+when relying on vendors who might otherwise fall short in their open
+source practice.
+
+Similarly, some vendors will be *unwilling* to participate in
+good faith.  That is, they will be willing to check boxes on the
+paperwork, but behaviorally will default to the type of closed
+development process typically found in government contracts.
+Sometimes this unwillingness comes from a lack of experience and open
+source skill.  In some other cases, perhaps, a vendor may hope the
+open source process fails, which could result in a final product that,
+while technically open source, cannot actually be released as open
+source, or that resists open source dynamics, thus leaving the vendor
+as the sole practical source for that component in the government
+procurement marketplace.
+
+Whatever the reason for failure to participate in open source process,
+agencys must have ways to bring recalcitrant vendors into process
+compliance.  Otherwise, the project risks failure across multiple
+teams.  To prepare for such situations, we recommend instituting Open
+Source Quality Assurance ("OSQA") at the project management level.
+
+OSQA is a set of practices designed to ensure that an open source
+project *behaves* like an open source project.  If vendors are
+delivering open source code but not engaging in the process, the
+software will hit some of its milestones but fail as collaboration
+falters over time.  Adding new vendors will be difficult.  Teams will
+discover API mismatches when they try to integrate modular pieces.
+Patches will go unreviewed, or worse, be accepted anyway.
+Documentation will degrade to the point that modules can only be
+deployed by the teams that developed them, which reintroduces vendor
+dependency and lock-in.  By the time these failures are obvious to the
+procuring agency, the final product will struggle as it experiences
+all the complexity of modular contracting without the benefits.
+
+There are a variety of OSQA techniques one can introduce.  OSQA "sits
+on the tree" and enforces standards at the pull request stage.  This
```

Comment:
I understand this is a draft, but it strikes me that this one sentence is way more technical than the rest of the text. Consider rephrasing this in more layman's terms